pkg/cli/cmd/radinit/recipe.go

```diff
@@ -52,148 +127,57 @@ func NewDevRecipeClient() DevRecipeClient {
 // GetDevRecipes is a function that queries a registry for recipes with a specific tag and returns a map of recipes.
 // If an error occurs, an error is returned.
 func (drc *devRecipeClient) GetDevRecipes(ctx context.Context) (map[string]map[string]corerp.RecipePropertiesClassification, error) {
-	reg, err := remote.NewRegistry(DevRecipesRegistry)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create client to registry %s -  %s", DevRecipesRegistry, err.Error())
-	}
+	logger := ucplog.FromContextOrDiscard(ctx)
```

Comment:
We don't use the logger for user-facing output in the CLI. Anything you log will be ignored. Please remove